caffe2/operators/experimental/c10/cpu/add_cpu.cc

```diff
@@ -69,11 +69,26 @@ void add_op_cpu_impl(
       C.mutable_data<DataType>(),
       static_cast<CPUContext*>(&context));
 }
+
+static auto registry = c10::RegisterOperators().op(
+    FunctionSchema(
```

Comment:
wait, does it mean that we lost ability to auto-deduce schemas?

or is it just to provide names?

Reply:
nevermind, it's in the following diff